interface/forms/eye_mag/SpectacleRx.php

```diff
@@ -343,34 +332,11 @@
                 font-size:12px;
             }
 
-            .right {
-                text-align: right;
-                vertical-align: middle;}
-
-            .left {
-                vertical-align: middle;
-                text-align: left;
-            }
-
             .title {
                 font-size: 0.9em;
                 font-weight: normal;
```

Comment:
@bradymiller is it just me or is this file completely broken in the eye mag form? xD

Reply:
@tywrenn , I was finally able to figure out how to even see this script, which looks fine:

![image](https://user-images.githubusercontent.com/278968/84213808-78eecc00-aa76-11ea-9aa4-a280525bbd76.png)

Reply:
Note my above screenshot was in most recent master. I'll need to test it on this PR also.
TODO for me

Reply:
@bradymiller If you can test that for me that would be great. The main thing that is a must for this file is to not have a horizontal scrollbar here. If there is one please let @stu01509 know so he can further investigate this.

Reply:
@stu01509 , @tywrenn , yep, there is a horizontal scrollbar:
![image](https://user-images.githubusercontent.com/278968/84248071-c4789880-aabd-11ea-9671-dafaeda9136e.png)

Reply:
Hi @bradymiller 

How can I open the Dispense RX modal?

Reply:
@sjpadgett For dlgopen, the only width and height that should be set is the iframe. The modal itself uses dynamic height attributes.

Reply:
it's a dialog. not a modal!

Reply:
**BS4 modal

Reply:
So, should I modify anything?